src/index.js

```diff
@@ -1,21 +1,17 @@
 "use strict";
 
-let WritableStream = require("./writable_stream");
+const WritableStream = require("./writable_stream");
 
-module.exports = class Renderer {
-	constructor(bundlePath) {
-		this.bundlePath = bundlePath;
-		this.reload();
-	}
+module.exports = bundle => {
+	const _render = bundle;
```

Comment:
I derived that from the code parts available before. Maybe bundle is not that good as param name so I will rename it